### PR TITLE
Improve TIN validator

### PR DESCRIPTION
**Pull request by FrancoOrtiz74**
This PR aims to add more verifications for the TIN validator

Fixes [BKO-3279](https://uphold.atlassian.net/browse/BKO-3279)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -11,13 +11,13 @@
 /**
  * Blacklist.
  */
-
-const blacklist = ['078051120', '219099999', '457555462'];
+const repitedNumbers = Array.from({ length: 9 - 0 + 1 }, (_, i) => i).map(current => String(current).repeat(9));
```
Comment (diogotorres97) [inline src/index.js]:
```suggestion
const repeatedNumbers = Array.from({ length: 10 }, (_, i) => i).map(current => String(current).repeat(9));
```

**Review by diogotorres97 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (afsampaio) [inline src/index.js]: Missing new line.

File: src/index.js [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
+const blacklist = ['078051120', '219099999', '457555462'].concat(repitedNumbers);
+const sequence = Array.from({ length: 9 - 0 + 1 }, (_, i) => i).reduce((acc, current) => acc + current, '').repeat(2);
```
Comment (diogotorres97) [inline src/index.js]:
```suggestion
const sequence = Array.from({ length: 10 }, (_, i) => i).reduce((acc, current) => acc + current, '').repeat(2);
```

Comment (diogotorres97) [reply src/index.js]: Missing descending sequences

Comment (diogotorres97) [reply src/index.js]: https://apidocs.taxbit.com/docs/certifying-tax-documentation#taxid-string-required

> The taxId is fully ascending or descending e.g. "123456789" or "987654321"
> The taxId is a single digit repeated e.g. "111111111"

File: src/index.js [hunk 3]
```diff
@@ -11,13 +11,13 @@
 /**
  * Blacklist.
  */
-
-const blacklist = ['078051120', '219099999', '457555462'];
+const repeatedNumbers = Array.from({ length: 10 }, (_, i) => i).map(current => String(current).repeat(9));
```
Comment (afsampaio) [inline src/index.js:15]: Please add new line.

Please add comment explaining the "limitations".

**Review by afsampaio — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 4]
```diff
@@ -8,16 +8,28 @@
  * See `http://www.irs.gov/Individuals/General-ITIN-Information` for more information.
  */
 
+/**
+ * Excludes repeated numbers as TIN e.g. 123456789.
```
Comment (diogotorres97) [inline src/index.js]:
```suggestion
 * Excludes repeated numbers as TIN e.g. 111111111.
```

**Pull request closed without merging**